ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -51,7 +51,7 @@ public protocol TwoLineTitleViewDelegate: AnyObject {
 // MARK: - TwoLineTitleView
 
 @objc(MSFTwoLineTitleView)
-open class TwoLineTitleView: UIView {
+open class TwoLineTitleView: UIView, TokenizedControlInternal {
```

Comment:
Since this is a UIView, there should be a `open override func willMove(toWindow newWindow: UIWindow?) {` somewhere.

Reply:
Pending the next batch of commits.

Reply:
This is consistent with our style guide: https://microsoft.github.io/swift-guide/FileOrganization.html#convention---constants